examples/chat/client/chat.cc

```diff
@@ -75,75 +74,73 @@ class Safe {
   T val_ GUARDED_BY(mu_);
 };
 
-void Prompt(std::shared_ptr<Safe<std::string>> room_name, const std::string& user_handle) {
-  std::cout << room_name->get() << ":" << user_handle << "> ";
-}
+void Prompt(const std::string& user_handle) { std::cout << user_handle << "> "; }
 
 void ListenLoop(Chat::Stub* stub, const RoomId& room_id, const std::string& user_handle,
```

Comment:
Nit: Is this a Loop because we have a while inside? I would feel more natural to call this OnReceive, OnSend or something.

Reply:
Yes, it's got `Loop` in the name because it does (and is run in a thread of its very own as a result). 

(For me, a function named `OnReceive` would imply something that's called by some external scheduler when a message is received.)